### ISSUE TITLE
Be able to start typing in the Recent Item Filter Box with out having to click in it first

### ISSUE THREAD

**Issue by yannduran** [luminous-software/start-page-plus#6]
### Description
When the Start Page+ window is visible, the 'Recent Items' filter doesn't have keyboard focus, so you have to click in it first.

### Steps to recreate
N/A

### Current behavior
You can't start typing immediately unless you've already clicked in it.

### Expected behavior
You should be able to start typing to filter the items immediately without having to click in it first.

**Comment by yannduran** [comment]
_v1.0_ will include this fix - see [Milestone v1.0](https://github.com/luminous-software/start-page-plus/milestone/5) for other features/fixes included in _v1.0_
